packages/ui-kit/src/components/globals.tsx

```diff
@@ -81,8 +81,9 @@ const Globals = (props: TGlobalsProps) => (
       kbd,
       samp,
       pre {
-        font-family: ${typography.fontFamilies.code}, 'Menlo', 'Monaco',
+        font-family: ${typography.fontFamilies.nonMonospace}, 'Menlo', 'Monaco',
           'Consolas', 'Liberation Mono', 'Courier New', monospace;
+        font-feature-settings: 'salt', 'ss02';
         font-size: ${typography.fontSizes.small};
```

Comment:
We could also think about adjusting the font-size here...the mime types look a bit small to me.

Reply:
obsolete after the new component for string literals without a box is there.